README.md

```diff
@@ -2,10 +2,43 @@
 
 The Node.js Foundation Technical Steering Committee is the technical governing body of the Node.js Foundation. It admits and oversees all top-level Projects in the Node.js Foundation. It also elects a representative to the Node.js Foundation Board of Directors.
 
+The TSC is community and contributor driven and is a home for community managed 
+projects and assets. The mission of the Node.js Foundation is to enable and encourage 
+the adoption of Node.js and the TSC is the home of the community driven portion
+of executing that mission.
+
+Since the Foundation and the TSC are intended to support the community the size and scope
```

Comment:
hrm... I don't think we can avoid using the word community entirely. while "contributor driven" describes part of the story we also are driven but the users of the work we do and make great effort to incorporate the needs of those users above the whims of contributors.